### ISSUE TITLE
Hidden 'Last modified' column doesn't work for new files

### ISSUE THREAD

**Issue by ShadowTemplate** [jupyterlab/jupyterlab#11041]
Good morning,

This morning I noticed that even if the 'Last modified' column is hidden, the date is displayed for newly created files.
For these files, the option is not honored and timestamps are visible.

You can find a simple demo [here](https://www.dropbox.com/s/uleqjcwixmchiv3/jupyterlab_bug_demo.mp4).

My environment is the following.

OS: CentOS 8
Browser: Chrome 93
Jupyter:
```
jupyter-client                    6.1.12             
jupyter-console                   6.4.0              
jupyter-core                      4.7.1              
jupyter-enterprise-gateway        2.5.0              
jupyter-kernel-gateway            2.5.0              
jupyter-server                    1.8.0              
jupyter-telemetry                 0.1.0              
jupyterhub                        1.4.1              
jupyterhub-ldapauthenticator      1.3.2              
jupyterlab                        3.0.16             
jupyterlab-pygments               0.1.2              
jupyterlab-server                 2.5.2  
```

Thanks a lot for your great work!

**Comment by krassowski** [comment]
Hi @ShadowTemplate, this is a duplicate of #8860 and that bug was fixed in #10100 which was released with JupyterLab 3.1. Please upgrade JupyterLab to have it working.

**Comment by ShadowTemplate** [comment]
Oh, sorry! I quickly searched for similar issues, but I didn't find anything.

Thanks a lot for your fast response!
Feel free to close this issue and have a great day.
